### PR TITLE
removes fucking holoparasites from the easiest to get slime in the fucking game

### PR DESCRIPTION
**Pull request by ghost**
Really?

Really?

I removed them from traitor, that doesn't mean 'give it to the job that already has the best shit in the world

You can already get legions of golems with bombs and summon hordes of murderous monsters to serve you, including magicarp, and make yourself a changeling.

We don't need to give them holoparasites too. Come on.

To make it worse these are the carp holoparasites which have no upper limit on how many you can have and lings can take them too.

I dunno about you but xenobiology coming out as flying skeleton changelings with holoparasites is a bit much, no?
##### Changelog

:cl:
rscadd: You can't get holoparasites from fucking xenobiology anymore (what)
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
Nice catch

**Comment by drderp3635** [comment]
I don't think these were ever something that was supposed to be obtained through xeno

**Comment by Cruix** [comment]
Leave a comment

**Comment by ShadowDeath6** [comment]
they actually were, but nobody ever got them because they're about a 1 in 150 chance to get. 

I doubt this is necessary, I haven't seen anybody get the item, including myself who actively tried to get it multiple rounds.

I also doubt adam saw it either until cruix pointed it out in his xeno pr, so I'm against this

tl;dr le kneejerk change that has no actual influence on the game. I'd prefer headslugs removed since that's twice as OP and twice as easy to get.

**Comment by doerakpoes** [comment]
Well geuss I won't be re trying xeno any time soon

**Review by X-TheDark — APPROVED** [review]
Compilation error is server-side, the PR is ok, codewise.

**Comment by ShadowDeath6** [comment]
I'd assume the number of people to have gotten holo from xenobio from the time this pr was made is less than 10

**Comment by ShadowDeath6** [comment]
sigh
